src/openapi/streaming/subscription.js

```diff
@@ -35,14 +35,22 @@ const LOG_AREA = 'Subscription';
 
 // -- Local methods section --
 
-function getUrl(url, args) {
-    if (!args.Top) {
+function getUrl(url, subscriptionData) {
+    if (!subscriptionData.Top) {
         return url;
     }
 
     return url+'?$top={Top}';

```

Comment:
I don't like it. Strange idea to drive pagination through url encoded parameters. @lukeapage What do you think about it? Maybe it's worth pushing openapi to just allow for passing Top parameters as part of standard arguments, next to Format and RefreshRate?